src/auth.js

```diff
@@ -0,0 +1,16 @@
+import {get,readable} from 'svelte/store';
+var value;
```

Comment:
You should provide an initial value for what state should return, even if in practice it gets set to something else instantaneously. I'm also a bit uncomfortable with using status codes here. Maybe just set this to the "LOADING", "FAILED", "SUCCEEDED" states we drew up?

Reply:
204 should be treated equally to 200, and doesn't indicate "PENDING". "PENDING" should be the default value of `value` before the auth fetch causes it to change. (i.e., change `var value;` above to `var value = "PENDING";`

Reply:
`amsa` might also return `403` instead of `401`, and that should be treated as a failure, also.